packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -622,11 +622,15 @@ class DevFS {
       final String packagePath = fs.path.fromUri(packageUri);
       final Directory packageDirectory = fs.directory(packageUri);
       Uri directoryUriOnDevice = fs.path.toUri(fs.path.join('packages', packageName) + fs.path.separator);
-      bool packageExists;
+      bool packageExists = packageDirectory.existsSync();
+
+      if (!packageExists) {
```

Comment:
Why not inline the check and since you're bailing out here, just assume `packageExists == true` in the code below?